mysql-client-tests/node/index.js

```diff
@@ -1,12 +1,12 @@
-let mysql = require('mysql');
+const mysql = require('mysql');
```

Comment:
js best practice is to use `const` where applicable... didn't necessarily see a hoisting prob in here, but `var` is like taboo js now